rebasehelper/constants.py

```diff
@@ -22,3 +22,4 @@
 
 PROGRAM_DESCRIPTION = "Tool to help package maintainers rebase their packages to the latest upstream version"
 NEW_ISSUE_LINK = 'https://github.com/rebase-helper/rebase-helper/issues/new'
+DEFAULT_CONFIG_LOCATIONS = ("$XDG_CONFIG_HOME", "$HOME/.config")
```

Comment:
As written above, this is not a good idea. Besides you have it wrong, it should be `"XDG_CONFIG_HOME"`, not `"$XDG_CONFIG_HOME"`.

You could define a constant for the resulting path (and move the expanding logic here).